### DISCUSSION TITLE
Make text look like title

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#884] [Ideas]
Many apps will combine native Wave objects with 3rd party graphing libraries. It would be nice to have a `ui.title` component which would be similar to one of the `ui.text` component for creating stand-alone titles for plots made outside of wave. 

Here is an example of using just regular `ui.text`:
<img width="1217" alt="image" src="https://user-images.githubusercontent.com/4793561/122656914-ca16ae80-d113-11eb-849d-b740388decf6.png">

or bolding the `ui.text`
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/4793561/122656951-09dd9600-d114-11eb-8f22-f8fee28b18ce.png">

**Comment by mturoci** [comment]
Agree, would be useful, converting into an issue.
